interceptors/logging/interceptors_test.go

```diff
@@ -110,11 +110,16 @@ func (l *baseMockLogger) Lines() []LogLine {
 
 type mockLogger struct {
 	*baseMockLogger
-
+	m      sync.Mutex
 	fields logging.Fields
 }
 
 func (l *mockLogger) Log(lvl logging.Level, msg string) {
+
```

Comment:
Could you remove this newline please :grin:?

Reply:
Sure :sunglasses: